app/Http/Controllers/InvitationController.php

```diff
@@ -12,6 +12,13 @@
 
 class InvitationController extends Controller
 {
+    public function showInvitations(): Response
+    {
+        // Fetch invitation data from the database
+        $invitations = Invitation::with('user')->get();
```

Comment:
Paginate results here , you expect to have hundreds of invitations listed here. Also any reason why the default datatable was not used ?

Reply:
Hey @muarachmann, I didn't understand what you mean by default database here. here we are using our invitation table and fetching the user details..

Reply:
There is vue datatable included in the project use it. See how users are displayed in the table.